lib/flutter_html.dart

```diff
@@ -73,15 +73,18 @@ class Html extends StatefulWidget {
     @required dom.Document? document,
     this.onLinkTap,
     this.onAnchorTap,
-    this.customRenders = const {},
+    this.extensions = const [],
     this.onCssParseError,
-    this.onImageError,
     this.shrinkWrap = false,
-    this.onImageTap,
-    this.tagsList = const [],
+    this.doNotRenderTheseTags,
+    this.onlyRenderTheseTags,
     this.style = const {},
   })  : data = null,
         assert(document != null),
+        assert(
+          onlyRenderTheseTags == null || doNotRenderTheseTags == null,
```

Comment:
Shouldn't this be `onlyRenderTheseTags != null && doNotRenderTheseTags != null`

Reply:
It's a bit unusual, but in this case, the user is allowed to provide neither of the two options, or one of them, but never both of them. This assertion guarantees that. Do you have any suggestions for how I could make the documentation on these two options clearer?

Reply:
To me it looks like the assertion asks me to provide both values. The exact opposite of what you want.

Reply:
Thinking about it a bit more, I don't think that it's an error to provide both values. I can't think up a use case where it would be necessary, but the widget works just fine with both configured. I'll remove the assertion.